portal/helpers/location.py

```diff
@@ -135,4 +145,22 @@ def get_location_from_api(payload):
     except ApiException as e:
         error = 'API error: %s' % e
 
-    return error, default_country, default_town, default_lat, default_lng
\ No newline at end of file
+    return error, default_country, default_town, default_lat, default_lng
+
+
+def get_uk_locations(postcodes):
+    payload = {'postcodes': postcodes}
+    headers = {'content-type': 'application/json'}
+    res = requests.post('http://api.postcodes.io/postcodes', data=json.dumps(payload), headers=headers)
```

Comment:
Wrap the requests.post() in an exception handler. See the previous function for exceptions thrown.